src/view/textproxy.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import Node from './node.js';
+
+/**
+ * Tree view text proxy.
```

Comment:
Some additional explanation would be useful. There should be explanation why TextProxy exists. Note that is can be broken if the text node changes. Note that it is not a node, it should not be inserted as a child.